src/app/accounting/migrate-opening-balances/migrate-opening-balances.component.html

```diff
@@ -80,15 +80,15 @@
 
             <div fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column" [formGroupName]="i">
 
-              <div fxFlex="8%">
+              <div fxFlex="8%" class="formField">
```

Comment:
use bootstarp classes instead of custom class
refer this https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
> use bootstrap classes instead of custom class
> refer this https://getbootstrap.com/docs/4.0/utilities/spacing/

Made changes. Please review